### PR TITLE
Add curly rule for statement after flow control clause

### PR DESCRIPTION
**Pull request by linlin1023**
- `curly`, these will fail:

```
if (foo) foo++;

while (bar)
    baz();

if (foo) {
    baz();
} else qux();
```

- `@typescript-eslint/indent` will fix:

```
interface DbInput<EntityType extends object> {
      tableName: string;
    }
```

### CODE REVIEW

**Review by dekpient — CHANGES_REQUESTED** [review]
this reminds me; we should add the sort import thingy as a dep @0608jennylee ?

**Comment by 0608jennylee** [comment]
> yup I've asked @linlin1023 to add it as well

File: configs/common.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ module.exports = {
       exports: "never",
       functions: "never"
     }],
-    "simple-import-sort/imports": "error"
+    "simple-import-sort/imports": "error",
```
Comment (cml35) [inline configs/common.js:64]: Do we want it as warn?

Comment (linlin1023) [reply configs/common.js:64]: we want error correct me if I get it wrong @dekpient , so far all the forced rules are set to be error level

Comment (0608jennylee) [reply configs/common.js:64]: It should be error, it should be changed to warn only for repos that we dont want to sort all the imports yet

**Review by 0608jennylee — APPROVED** [review]

**Review by cojj90 — DISMISSED** [review]
lgtm

**Review by cojj90 — APPROVED** [review]

**Review by dekpient — APPROVED** [review]

**Review by cojj90 — APPROVED** [review]

**Pull request merged**